src/wp-includes/post.php

```diff
@@ -1757,6 +1763,8 @@ function get_post_type_labels( $post_type_object ) {
 		'item_reverted_to_draft'   => array( __( 'Post reverted to draft.' ), __( 'Page reverted to draft.' ) ),
 		'item_scheduled'           => array( __( 'Post scheduled.' ), __( 'Page scheduled.' ) ),
 		'item_updated'             => array( __( 'Post updated.' ), __( 'Page updated.' ) ),
+		'item_link'                => array( null, null ),
+		'item_link_description'    => array( null, null ),
```

Comment:
I left these as null, since it was a bit odd to have the other CPT/taxonomies default to something like 'A link to a post'. Let me know if I should remove this or have post/page defaults.

As an aside, generally how quickly do plugins notice new labels to add to their CPT or taxonomy definitions?

Reply:
I would provide a default.

How quickly plugins adapt features really varies. This is a search for the most recently added CPT label: https://wpdirectory.net/search/01EZJCAVRHVFSNJE5RWGSRS2XV Looking at that the adoption is fairly low. The major plugins have as well as the plugins that help people create their own CPTs. 

I would work on the assumption that developers won't update immediately. Also to keep in mind people who have had custom websites developed may not ever have their CPT labels updated depending on how on top of it they are.

Reply:
Thanks @TimothyBJacobs ! Updated in 5ba189a. 

This one will be interesting to balance. I am tempted to provide an is_singular fallback but I also think doing so will cause this label to not be spotted by developers. Here's an example where I installed a Portfolio plugin.

<img width="1630" alt="Screen Shot 2021-03-01 at 8 58 26 AM" src="https://user-images.githubusercontent.com/1270189/109533267-e3b1be80-7a6e-11eb-965e-e6035844df7e.png">